src/shared/components/donate.tsx

```diff
@@ -82,10 +82,13 @@ const DonateButtons = () => (
 );
 
 const FundingGoal = () => {
-  const percent_funded = FUNDED_DEVS / FUNDED_DEV_GOAL;
+  var percent_funded = FUNDED_DEVS / FUNDED_DEV_GOAL;
+  percent_funded = Math.round(percent_funded * 100);
+  percent_funded = Math.min(percent_funded, 100);
```

Comment:
`const percentFunded = Math.min(Math.round(FUNDED_DEVS / FUNDED_DEV_GOAL*100), 100));`

Reply:
Seems a bit harder to read but okay.